### ISSUE TITLE
How can I get the HD map information

### ISSUE THREAD

**Issue by InterKnight** [Tsinghua-MARS-Lab/InterSim#9]
Hi, Thank you for great work ! 
I'm wondering if the planning algorithm need HD map, how can I get the map information?
I need get map data from WOMD/NuPlan and convert data format to meet algorithm requirements？

**Comment by larksq** [comment]
Yes, the planning algorithm uses HD map. You can find the converted data format in the road_dic. All HD map information is packed into this dictionary for the planner to use.
